doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -738,6 +739,8 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
   - `path_to_library` uses an OpenSSL v1.1.x library that's already built.  You can use `${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr` as `path_to_library` when you are configuring in the Docker container.
   - Using `--with-openssl=fetched` will fail during the build in the Docker environment.
 
+:pencil: **DDR support:** Currently you can build DDR support only on AArch64 Linux.  If you are building in the Docker container, you need the `--disable-ddr` option.
```

Comment:
This comment doesn't actually mention cross compiling, I'm thinking it should. Is it a problem if the docker container is on an AArch64 Linux machine?

Reply:
No particular reason.
I changed "building in the Docker container" to "building in a cross-compilation environment".